executor/api/rest/client.go

```diff
@@ -139,24 +142,26 @@ func (smc *JSONRestClient) getMetricsRoundTripper(modelName string, service stri
 
 func (smc *JSONRestClient) addHeaders(req *http.Request, m map[string][]string) {
 	for k, vv := range m {
-		for _, v := range vv {
-			req.Header.Set(k, v)
+		if _, ok := headersIgnore[k]; !ok {
+			for _, v := range vv {
+				req.Header.Set(k, v)
+			}
 		}
 	}
 }
 
-func (smc *JSONRestClient) doHttp(ctx context.Context, modelName string, method string, url *url.URL, msg []byte, meta map[string][]string) ([]byte, string, error) {
+func (smc *JSONRestClient) doHttp(ctx context.Context, modelName string, method string, url *url.URL, msg []byte, meta map[string][]string, contentType string) ([]byte, string, error) {
 	smc.Log.Info("Calling HTTP", "URL", url)
 
 	var req *http.Request
 	var err error
 	if msg != nil {
-		smc.Log.Info("Building message")
+		smc.Log.Info("Building message", "contentType", contentType, "msg", string(msg))
```

Comment:
Is it possible to ignore `INFO` level in the logs atm? I'm just wondering whether we should remove `msg` until we confirm that point (to avoid making it extra verbose).

Reply:
I'll remove these